cmd/ethrex/ethrex.rs

```diff
@@ -208,6 +208,20 @@ async fn main() {
 
     tracker.spawn(rpc_api);
 
+    // Start the prometheus /metrics api
+    cfg_if::cfg_if! {
```

Comment:
does it make sense to have this as a feature flag? I would think that if we have it as a cli arg then we should enable/disable it based on that.

Reply:
Right, i was thinking in conditionally building the metrics crate based on the `metrics` feature, we can build the metrics api and then enable it only if the port was set. I'm working on it.

Reply:
Fixed in [1d45ecdc](https://github.com/lambdaclass/ethrex/pull/1518/commits/1d45ecdcd8bbcf8a6226fd5d65a28a52868dcbdc)